Sourcing minigraphs...

A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.1.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/batch/../src/miniaturize.py", line 17, in <module>
    from minigraphs import Metropolis
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/__init__.py", line 7, in <module>
    from minigraphs.Metropolis import *
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/Metropolis.py", line 10, in <module>
    import pandas as pd
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/api.py", line 28, in <module>
    from pandas.core.arrays import Categorical
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/__init__.py", line 1, in <module>
    from pandas.core.arrays.arrow import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/__init__.py", line 5, in <module>
    from pandas.core.arrays.arrow.array import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/array.py", line 50, in <module>
    from pandas.core import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/ops/__init__.py", line 8, in <module>
    from pandas.core.ops.array_ops import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/ops/array_ops.py", line 56, in <module>
    from pandas.core.computation import expressions
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/computation/expressions.py", line 21, in <module>
    from pandas.core.computation.check import NUMEXPR_INSTALLED
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/computation/check.py", line 5, in <module>
    ne = import_optional_dependency("numexpr", errors="warn")
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/site-packages/numexpr/__init__.py", line 24, in <module>
    from numexpr.interpreter import MAX_THREADS, use_vml, __BLOCK_SIZE1__

A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.1.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/batch/../src/miniaturize.py", line 17, in <module>
    from minigraphs import Metropolis
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/__init__.py", line 7, in <module>
    from minigraphs.Metropolis import *
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/Metropolis.py", line 10, in <module>
    import pandas as pd
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/api.py", line 28, in <module>
    from pandas.core.arrays import Categorical
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/__init__.py", line 1, in <module>
    from pandas.core.arrays.arrow import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/__init__.py", line 5, in <module>
    from pandas.core.arrays.arrow.array import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/array.py", line 50, in <module>
    from pandas.core import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/ops/__init__.py", line 8, in <module>
    from pandas.core.ops.array_ops import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/ops/array_ops.py", line 56, in <module>
    from pandas.core.computation import expressions
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/computation/expressions.py", line 21, in <module>
    from pandas.core.computation.check import NUMEXPR_INSTALLED
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/computation/check.py", line 5, in <module>
    ne = import_optional_dependency("numexpr", errors="warn")
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/site-packages/numexpr/__init__.py", line 24, in <module>
    from numexpr.interpreter import MAX_THREADS, use_vml, __BLOCK_SIZE1__
AttributeError: _ARRAY_API not found

A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.1.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/batch/../src/miniaturize.py", line 17, in <module>
    from minigraphs import Metropolis
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/__init__.py", line 7, in <module>
    from minigraphs.Metropolis import *
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/Metropolis.py", line 10, in <module>
    import pandas as pd
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/api.py", line 28, in <module>
    from pandas.core.arrays import Categorical
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/__init__.py", line 1, in <module>
    from pandas.core.arrays.arrow import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/__init__.py", line 5, in <module>
    from pandas.core.arrays.arrow.array import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/array.py", line 50, in <module>
    from pandas.core import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/ops/__init__.py", line 8, in <module>
    from pandas.core.ops.array_ops import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/ops/array_ops.py", line 56, in <module>
    from pandas.core.computation import expressions
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/computation/expressions.py", line 21, in <module>
    from pandas.core.computation.check import NUMEXPR_INSTALLED
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/computation/check.py", line 5, in <module>
    ne = import_optional_dependency("numexpr", errors="warn")
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/site-packages/numexpr/__init__.py", line 24, in <module>
    from numexpr.interpreter import MAX_THREADS, use_vml, __BLOCK_SIZE1__

A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.1.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/batch/../src/miniaturize.py", line 17, in <module>
    from minigraphs import Metropolis
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/__init__.py", line 7, in <module>
    from minigraphs.Metropolis import *
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/Metropolis.py", line 10, in <module>
    import pandas as pd
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/api.py", line 28, in <module>
    from pandas.core.arrays import Categorical
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/__init__.py", line 1, in <module>
    from pandas.core.arrays.arrow import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/__init__.py", line 5, in <module>
    from pandas.core.arrays.arrow.array import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/array.py", line 50, in <module>
    from pandas.core import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/ops/__init__.py", line 8, in <module>
    from pandas.core.ops.array_ops import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/ops/array_ops.py", line 56, in <module>
    from pandas.core.computation import expressions
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/computation/expressions.py", line 21, in <module>
    from pandas.core.computation.check import NUMEXPR_INSTALLED
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/computation/check.py", line 5, in <module>
    ne = import_optional_dependency("numexpr", errors="warn")
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/site-packages/numexpr/__init__.py", line 24, in <module>
    from numexpr.interpreter import MAX_THREADS, use_vml, __BLOCK_SIZE1__
AttributeError: _ARRAY_API not found
AttributeError: _ARRAY_API not found
AttributeError: _ARRAY_API not found

A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.1.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/batch/../src/miniaturize.py", line 17, in <module>
    from minigraphs import Metropolis
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/__init__.py", line 7, in <module>
    from minigraphs.Metropolis import *
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/Metropolis.py", line 10, in <module>
    import pandas as pd
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/api.py", line 28, in <module>
    from pandas.core.arrays import Categorical
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/__init__.py", line 1, in <module>
    from pandas.core.arrays.arrow import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/__init__.py", line 5, in <module>
    from pandas.core.arrays.arrow.array import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/array.py", line 50, in <module>
    from pandas.core import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/ops/__init__.py", line 8, in <module>
    from pandas.core.ops.array_ops import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/ops/array_ops.py", line 56, in <module>
    from pandas.core.computation import expressions
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/computation/expressions.py", line 21, in <module>
    from pandas.core.computation.check import NUMEXPR_INSTALLED
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/computation/check.py", line 5, in <module>
    ne = import_optional_dependency("numexpr", errors="warn")
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/site-packages/numexpr/__init__.py", line 24, in <module>
    from numexpr.interpreter import MAX_THREADS, use_vml, __BLOCK_SIZE1__

A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.1.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/batch/../src/miniaturize.py", line 17, in <module>
    from minigraphs import Metropolis
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/__init__.py", line 7, in <module>
    from minigraphs.Metropolis import *
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/Metropolis.py", line 10, in <module>
    import pandas as pd
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/api.py", line 28, in <module>
    from pandas.core.arrays import Categorical
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/__init__.py", line 1, in <module>
    from pandas.core.arrays.arrow import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/__init__.py", line 5, in <module>
    from pandas.core.arrays.arrow.array import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/array.py", line 50, in <module>
    from pandas.core import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/ops/__init__.py", line 8, in <module>
    from pandas.core.ops.array_ops import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/ops/array_ops.py", line 56, in <module>
    from pandas.core.computation import expressions
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/computation/expressions.py", line 21, in <module>
    from pandas.core.computation.check import NUMEXPR_INSTALLED
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/computation/check.py", line 5, in <module>
    ne = import_optional_dependency("numexpr", errors="warn")
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/site-packages/numexpr/__init__.py", line 24, in <module>
    from numexpr.interpreter import MAX_THREADS, use_vml, __BLOCK_SIZE1__
AttributeError: _ARRAY_API not found
AttributeError: _ARRAY_API not found

A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.1.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/batch/../src/miniaturize.py", line 17, in <module>
    from minigraphs import Metropolis
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/__init__.py", line 7, in <module>
    from minigraphs.Metropolis import *
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/Metropolis.py", line 10, in <module>
    import pandas as pd
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/api.py", line 28, in <module>
    from pandas.core.arrays import Categorical
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/__init__.py", line 1, in <module>
    from pandas.core.arrays.arrow import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/__init__.py", line 5, in <module>
    from pandas.core.arrays.arrow.array import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/array.py", line 64, in <module>
    from pandas.core.arrays.masked import BaseMaskedArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/masked.py", line 60, in <module>
    from pandas.core import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/nanops.py", line 52, in <module>
    bn = import_optional_dependency("bottleneck", errors="warn")
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/site-packages/bottleneck/__init__.py", line 7, in <module>
    from .move import (move_argmax, move_argmin, move_max, move_mean, move_median,

A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.1.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/batch/../src/miniaturize.py", line 17, in <module>
    from minigraphs import Metropolis
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/__init__.py", line 7, in <module>
    from minigraphs.Metropolis import *
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/Metropolis.py", line 10, in <module>
    import pandas as pd
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/api.py", line 28, in <module>
    from pandas.core.arrays import Categorical
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/__init__.py", line 1, in <module>
    from pandas.core.arrays.arrow import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/__init__.py", line 5, in <module>
    from pandas.core.arrays.arrow.array import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/array.py", line 64, in <module>
    from pandas.core.arrays.masked import BaseMaskedArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/masked.py", line 60, in <module>
    from pandas.core import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/nanops.py", line 52, in <module>
    bn = import_optional_dependency("bottleneck", errors="warn")
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/site-packages/bottleneck/__init__.py", line 7, in <module>
    from .move import (move_argmax, move_argmin, move_max, move_mean, move_median,

A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.1.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/batch/../src/miniaturize.py", line 17, in <module>
    from minigraphs import Metropolis
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/__init__.py", line 7, in <module>
    from minigraphs.Metropolis import *
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/Metropolis.py", line 10, in <module>
    import pandas as pd
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/api.py", line 28, in <module>
    from pandas.core.arrays import Categorical
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/__init__.py", line 1, in <module>
    from pandas.core.arrays.arrow import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/__init__.py", line 5, in <module>
    from pandas.core.arrays.arrow.array import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/array.py", line 64, in <module>
    from pandas.core.arrays.masked import BaseMaskedArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/masked.py", line 60, in <module>
    from pandas.core import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/nanops.py", line 52, in <module>
    bn = import_optional_dependency("bottleneck", errors="warn")
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/site-packages/bottleneck/__init__.py", line 7, in <module>
    from .move import (move_argmax, move_argmin, move_max, move_mean, move_median,
AttributeError: 
A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.1.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/batch/../src/miniaturize.py", line 17, in <module>
    from minigraphs import Metropolis
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/__init__.py", line 7, in <module>
    from minigraphs.Metropolis import *
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/Metropolis.py", line 10, in <module>
    import pandas as pd
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/api.py", line 28, in <module>
    from pandas.core.arrays import Categorical
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/__init__.py", line 1, in <module>
    from pandas.core.arrays.arrow import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/__init__.py", line 5, in <module>
    from pandas.core.arrays.arrow.array import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/array.py", line 64, in <module>
    from pandas.core.arrays.masked import BaseMaskedArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/masked.py", line 60, in <module>
    from pandas.core import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/nanops.py", line 52, in <module>
    bn = import_optional_dependency("bottleneck", errors="warn")
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/site-packages/bottleneck/__init__.py", line 7, in <module>
    from .move import (move_argmax, move_argmin, move_max, move_mean, move_median,
AttributeError: _ARRAY_API not found
_ARRAY_API not found
AttributeError: _ARRAY_API not found
AttributeError: _ARRAY_API not found

A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.1.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/batch/../src/miniaturize.py", line 17, in <module>
    from minigraphs import Metropolis
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/__init__.py", line 7, in <module>
    from minigraphs.Metropolis import *
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/Metropolis.py", line 10, in <module>
    import pandas as pd
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/api.py", line 28, in <module>
    from pandas.core.arrays import Categorical
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/__init__.py", line 1, in <module>
    from pandas.core.arrays.arrow import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/__init__.py", line 5, in <module>
    from pandas.core.arrays.arrow.array import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/array.py", line 64, in <module>
    from pandas.core.arrays.masked import BaseMaskedArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/masked.py", line 60, in <module>
    from pandas.core import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/nanops.py", line 52, in <module>
    bn = import_optional_dependency("bottleneck", errors="warn")
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/site-packages/bottleneck/__init__.py", line 7, in <module>
    from .move import (move_argmax, move_argmin, move_max, move_mean, move_median,
AttributeError: _ARRAY_API not found

A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.1.1 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

Traceback (most recent call last):  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/batch/../src/miniaturize.py", line 17, in <module>
    from minigraphs import Metropolis
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/__init__.py", line 7, in <module>
    from minigraphs.Metropolis import *
  File "/mnt/ufs18/home-245/martjor/repos/dev_pt/src/minigraphs/Metropolis.py", line 10, in <module>
    import pandas as pd
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/__init__.py", line 49, in <module>
    from pandas.core.api import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/api.py", line 28, in <module>
    from pandas.core.arrays import Categorical
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/__init__.py", line 1, in <module>
    from pandas.core.arrays.arrow import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/__init__.py", line 5, in <module>
    from pandas.core.arrays.arrow.array import ArrowExtensionArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/arrow/array.py", line 64, in <module>
    from pandas.core.arrays.masked import BaseMaskedArray
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/arrays/masked.py", line 60, in <module>
    from pandas.core import (
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/core/nanops.py", line 52, in <module>
    bn = import_optional_dependency("bottleneck", errors="warn")
  File "/mnt/home/martjor/.local/lib/python3.11/site-packages/pandas/compat/_optional.py", line 135, in import_optional_dependency
    module = importlib.import_module(name)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/mnt/home/martjor/anaconda3/lib/python3.11/site-packages/bottleneck/__init__.py", line 7, in <module>
    from .move import (move_argmax, move_argmin, move_max, move_mean, move_median,
AttributeError: _ARRAY_API not found
Miniaturizing 'soc-pages-tvshow' to size 300...
Beta opt: 67.29724079125445

	 - Target metrics: {'density': 0.00227, 'assortativity_norm': 0.78, 'clustering': 0.37}
	 - Weights: {'density': 125.94405388215176, 'assortativity_norm': 3.3803303760761745, 'clustering': 2.7466993829082047}
	 - Functions: {'density': <function density at 0x14c00423b7e0>, 'assortativity_norm': <function <lambda> at 0x14c01d5247c0>, 'clustering': <function average_clustering at 0x14bff5f8b240>}

Cycle 1/5 <<<

Iteration 1/20

Iteration 2/20

Iteration 3/20

Iteration 4/20

Iteration 5/20

Iteration 6/20

Iteration 7/20

Iteration 8/20

Iteration 9/20

Iteration 10/20

Iteration 11/20

Iteration 12/20

Iteration 13/20

Iteration 14/20

Iteration 15/20

Iteration 16/20

Iteration 17/20

Iteration 18/20

Iteration 19/20

Iteration 20/20


Cycle 2/5 <<<

Iteration 1/20

Iteration 2/20

Iteration 3/20

Iteration 4/20

Iteration 5/20

Iteration 6/20

Iteration 7/20

Iteration 8/20

Iteration 9/20

Iteration 10/20

Iteration 11/20

Iteration 12/20

Iteration 13/20

Iteration 14/20

Iteration 15/20

Iteration 16/20

Iteration 17/20

Iteration 18/20

Iteration 19/20

Iteration 20/20


Cycle 3/5 <<<

Iteration 1/20

Iteration 2/20

Iteration 3/20

Iteration 4/20

Iteration 5/20

Iteration 6/20

Iteration 7/20

Iteration 8/20

Iteration 9/20

Iteration 10/20

Iteration 11/20

Iteration 12/20

Iteration 13/20

Iteration 14/20

Iteration 15/20

Iteration 16/20

Iteration 17/20

Iteration 18/20

Iteration 19/20

Iteration 20/20


Cycle 4/5 <<<

Iteration 1/20

Iteration 2/20

Iteration 3/20

Iteration 4/20

Iteration 5/20

Iteration 6/20

Iteration 7/20

Iteration 8/20

Iteration 9/20

Iteration 10/20

Iteration 11/20

Iteration 12/20

Iteration 13/20

Iteration 14/20

Iteration 15/20

Iteration 16/20

Iteration 17/20

Iteration 18/20

Iteration 19/20

Iteration 20/20


Cycle 5/5 <<<

Iteration 1/20

Iteration 2/20

Iteration 3/20

Iteration 4/20

Iteration 5/20

Iteration 6/20

Iteration 7/20

Iteration 8/20

Iteration 9/20

Iteration 10/20

Iteration 11/20

Iteration 12/20

Iteration 13/20

Iteration 14/20

Iteration 15/20

Iteration 16/20

Iteration 17/20

Iteration 18/20

Iteration 19/20

Iteration 20/20

